plugins/menu/plugin.js

```diff
@@ -597,6 +600,39 @@ CKEDITOR.plugins.add( 'menu', {
  * @cfg {String} [menu_groups=see source]
  * @member CKEDITOR.config
  */
+
+/**
+ * Optional callback function that is called when menu is hidden.
+ *
+ * @member CKEDITOR.menu
+ * @method onHide
+ */
+
+/**
```

Comment:
Instead of documenting it this way, it would be better to mark `CKEDITOR.menu` class with `@mixins CKEDITOR.event`.